js/privatebin.js

```diff
@@ -43,10 +43,21 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
         var me = {};
 
         /**
+         * list of UserAgents (parts) known to belong to a bot
+         *
+         * @private
+         * @enum   {Object}
+         * @readonly
+         */
+        var BadBotUA = [
+            'Bot',
+            'bot'
+        ];
+
+        /**
          * character to HTML entity lookup table
          *
          * @see    {@link https://github.com/janl/mustache.js/blob/master/mustache.js#L60}
-         * @name Helper.entityMap
```

Comment:
The default JSDoc run as documented in the README will not document private methods and variables. But adding the option `-p` or `--access all` will include these private symbols. Its not ultra urgent but ideally I would like to have the option to have these documented, too.